src/components/Message/MessageItems.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const StyledMessageItems = styled.div`
+  background: #fff;
+  max-width: 400px;
+  div:nth-child(1) {
```

Comment:
what does this do?
the first child of each `div` under `StyledMessageItems` has to have no border?